src/bsk_rl/envs/general_satellite_tasking/scenario/satellites.py

```diff
@@ -294,7 +294,7 @@ class AccessSatellite(Satellite):
     def __init__(
         self,
         *args,
-        generation_duration: float = 60 * 95,
+        generation_duration: float = 600.0,
```

Comment:
Should it be 6,000.0 instead of 600.0 to be closer to 60*95?

Reply:
No, 6000 is excessive. Usually what happens is you calculate up to your time limit, but at the end you just need a handful of extra observations past the time limit to fill out your observation. If you needed to go past 600s, it would calculate additional 600s increments until it has everything it needs.